### ISSUE TITLE
Any new tab that pops up, ask to close it

### ISSUE THREAD

**Issue by jddcef** [truefedex/tv-bro#208]
If a site manages to open a new tab, i.e. a popup occurs, which isn't user initiated with the new tab button or other menu.
Then please show a big red close button that I can press immediately.
So I can close popups quickly with one press.
I know false popups get through sometimes, but at least I want to close it easily. At the moment I must press back, down and then enter to close.  3 actions.
Maybe it can become 2 or 1 clicks.

**Comment by dryfish09** [comment]
Good idea:)

**Comment by tseugtseug** [comment]
This is very necessary, since completely blocking pop-up tabs can be too drastic. Some sites offer what we need in a new tab, but there are also tabs with annoying ads. Generally, a good tab has the same domain as the site we’re browsing. Sometimes it's also necessary to jump to a different page. A checkbox to always allow certain new tabs, based on the domain, would be useful.

In short, it would be helpful to have a confirmation box to either accept or dismiss a pop-up tab (the link text should be visible so we can make an informed decision).

**Comment by jddcef** [comment]
Yeah all browsers should start doing this.  Any time a new tab is opened without the user initiating it with a new tab button, then there should be a way to close it immediately, with the default being chosen by the user (default to always continue from this site I was on or close all new tabs by default when I'm on this site, or ask (with default either dismiss or default continue))

**Comment by dryfish09** [comment]
I think repository author js busy, so we will dont have it soon. But it still good feature that any browsers must have
